src/realm/list.cpp

```diff
@@ -678,6 +690,84 @@ void Lst<Mixed>::add_index(Path& path, Index index) const
     path.emplace_back(ndx);
 }
 
+bool Lst<Mixed>::nullify(ObjLink link)
+{
+    size_t ndx = find_first(link);
+    if (ndx != realm::not_found) {
+        if (Replication* repl = Base::get_replication()) {
+            repl->list_erase(*this, ndx); // Throws
```

Comment:
Does this replication instruction have the correct path for a nested collection? Or is that coming soon?

Reply:
You can get the path from the list itself. The list is passed as parameter.

Reply:
Oh, that's convenient 👍